### PR TITLE
New version: PrismLauncher.PrismLauncher version 7.2

### PR DESCRIPTION
**Pull request by PrismAutomata**
###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/113214&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/p/PrismLauncher/PrismLauncher/7.2/PrismLauncher.PrismLauncher.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+InstallerType: nullsoft
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2023-07-26
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/PrismLauncher/PrismLauncher/releases/download/7.2/PrismLauncher-Windows-MSVC-Setup-7.2.exe
```
Comment (Scrumplex) [inline manifests/p/PrismLauncher/PrismLauncher/7.2/PrismLauncher.PrismLauncher.installer.yaml:15]: needs to be MSVC-Legacy

**Comment by stephengillie** [comment]
I do not know if the @PrismaAutomata account is monitored.

**Comment by Scrumplex** [comment]
> I do not know if the @PrismAutomata account is monitored.

I get emails for that account :D

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**